### PR TITLE
Support xvideos multidomain

### PR DESCRIPTION
**Pull request by siddht4**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [ ] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [ ] Read [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site)
- [ x] Read [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) and adjusted the code to meet them
- [ x] Covered the code with tests (note that PRs without tests will be REJECTED)
- [ x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [ ] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x ] Improvement
- [ ] New extractor
- [ x] New feature

---

### Description of your *pull request* and other information

Explanation of your *pull request* in arbitrary form goes here. Please make sure the description explains the purpose and effect of your *pull request* and is worded well enough to be understood. Provide as much context and examples as possible.

Multi domain support of xvideos

Can download or view xvideos through multi domain I.e

xvideos
xvideos2
xvideos3
xvideos4
xvideos5

Check passes class 38 in common.py 
And then attempts to download using multiple domains

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install flake8
+      run: pip install flake8
+    - name: Run flake8
+      run: flake8 .
```
Comment (dstftw) [inline .github/workflows/main.yml:81]: Remove all unrelated code.

File: youtube_dl/extractor/xvideos.py [hunk 2]
```diff
@@ -14,10 +14,11 @@
 
 
 class XVideosIE(InfoExtractor):
+
```
Comment (dstftw) [inline youtube_dl/extractor/xvideos.py:17]: Remove all unrelated changes.

File: youtube_dl/extractor/xvideos.py [hunk 3]
```diff
@@ -76,14 +77,31 @@ class XVideosIE(InfoExtractor):
...
     }]
-
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        webpage = self._download_webpage(
-            'https://www.xvideos.com/video%s/' % video_id, video_id)
-
+        self.report_download_webpage(video_id)
+        Host_List = ['xvideos', 'xvideos2', 'xvideos3','xvideos4','xvideos5']
```
Comment (dstftw) [inline youtube_dl/extractor/xvideos.py:98]: flake8.

File: youtube_dl/extractor/xvideos.py [hunk 4]
```diff
@@ -76,14 +77,31 @@ class XVideosIE(InfoExtractor):
...
+        for host in Host_List:
+            webpage = self._download_webpage(
+                'https://www.' + host + '.com/video%s/' % video_id, video_id,note=False, fatal=host==Host_List[-1], errnote=host==Host_List[-1])
```
Comment (dstftw) [inline youtube_dl/extractor/xvideos.py:101]: flake8.

**Review by dstftw — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (siddht4) [reply .github/workflows/main.yml:81]: Sure I did not realize my pull request was auto updated

**Comment by siddht4** [comment]
Closed as fresh code pushed
